app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt

```diff
@@ -50,6 +50,9 @@ interface ThreadViewModel {
 
         fun isFetchingReplies(): Observable<Boolean>
         fun loadMoreReplies(): Observable<Void>
+
+        /** Display the bottom pagination Error Cell **/
```

Comment:
On threads is displayed on top, modify the comment

Reply:
Done